CSS/css_completions.py

```diff
@@ -426,7 +426,7 @@ class CSSCompletions(sublime_plugin.EventListener):
     regex = None
 
     def on_query_completions(self, view, prefix, locations):
-        selector_scope = "source.css - meta.selector.css"
+        selector_scope = "source.css - meta.selector.css, text.html meta.attribute-with-value.style.html string.quoted punctuation.definition.string.end.html"
```

Comment:
In that case, shouldn't it be:

```
text.html meta.attribute-with-value.style.html string.quoted -punctuation.definition.string.start.html
```

So that we get completions for the whole string, but not before the opening quote?